lib/stateTransition/validation/validateStateTransitionStructureFactory.js

```diff
@@ -9,7 +9,7 @@ const InvalidStateTransitionTypeError = require('../../errors/InvalidStateTransi
 const StateTransitionMaxSizeExceededError = require('../../errors/StateTransitionMaxSizeExceededError');
 const MaxEncodedBytesReachedError = require('../../util/errors/MaxEncodedBytesReachedError');
 
-const baseSchema = require('../../../schema/stateTransition/base');
+const baseSchema = require('../../../schema/stateTransition/stateTransitionBase');
```

Comment:
Why not just `/base` as you've done with `documentTransition`?

Reply:
There are can be other schemas so it might be some confusion what the base of, isn't it?

For example, in identities, we have `identity` and `publicKey`. If we call it just base than it's not really clear I fell.

Reply:
Oh, you're right, my bad